transforms/images/polus-ftl-label-plugin/plugin.json

```diff
@@ -1,45 +1,45 @@
 {
     "name": "FTL Label",
-    "version": "0.2.2",
+    "version": "0.3.8",
     "title": "FTL Label",
     "description": "Label objects in a 2d or 3d binary image.",
-    "author": "Nick Schaub (nick.schaub@nih.gov)",
+    "authors": "Nick Schaub (nick.schaub@nih.gov), Najib Ishaq (najib.ishaq@axleinfo.com)",
```

Comment:
This field must be `author`, not `authors`.

Reply:
Fixed.